src/v1/services/app/post/index.js

```diff
@@ -43,4 +43,11 @@ router.get('/', validate(getPostsSchema), ({ query }, res, next) => {
     .catch(error => next(error));
 });
 
+router.delete('/:postId', validate(removePostSchema), ({ params }, res, next) => {
+  const { postId } = params;
+  removePost(postId)
+    .then(payload => res.status(200).json({ payload }))
```

Comment:
Desculpa o que ela faz é claro, o que ela retorna que não é claro. To no celular e não tem edital, Sorry